supermarq-benchmarks/supermarq/qcvv/irb.py

```diff
@@ -351,6 +358,8 @@ def plot_results(self) -> None:
             alpha=0.5,
             color="tab:orange",
         )
+        if filename is not None:
+            plt.savefig(filename)
```

Comment:
fwiw locally i usually have to use
```suggestion
            plt.savefig(filename, bbox_inches="tight")
```
to prevent weird margins/cutoffs around labels/etc. Not sure if it makes sense to use here though?

Reply:
also does this work?
```suggestion
            figure = plot.get_figure()
            figure.savefig(filename, bbox_inches="tight"))
```
(to avoid relying on the global mpl state)

Reply:
> also does this work?
> 
> (to avoid relying on the global mpl state)

Good shout